spec/features/work_packages/new_work_package_spec.rb

```diff
@@ -46,8 +46,8 @@
 
         work_packages_page.visit_new
 
-        input = page.find('#work_package_start_date')
-        input.click
+        # Fill in the date, a sa simple click does not seem to trigger the datepicker here
+        fill_in "Start date", with: DateTime.now.strftime('%Y-%m-%d')
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.